core/dyn-configs/README.md

```diff
@@ -1,5 +1,17 @@
-Dynamic config  helpers for the NEAR codebase.
+Dynamic config helpers for the NEAR codebase.
 
-This crate contains all utilities to dynamic control neard.
+This crate contains utilities that allow to reconfigure the node while it is running.
 
-- `EXPECTED_SHUTDOWN_AT`: the specified block height neard will gracefully shutdown at.
+## How to:
+
+### Logging and tracing
+
+Make changes to `log_config.json` and send `SIGHUP` signal to the `neard` process.
+
+### Other config values
+
+Makes changes to `config.json` and send `SIGHUP` signal to the `neard` process.
+
+#### Fields of config that can be changed while the node is running:
+
+- `expected_shutdown`: the specified block height neard will gracefully shutdown at.
```

Comment:
Could you add more comments here for the expected behavior if the user pollutes the config file by modifying a field that is not included here or change the config file to be invalid?